repair project

blackjack4494 opened the pull request:
angular websockets have been updated to newer version.
readme got updated (may have to change repo again).
changed way of identification of mtr link

app works again.

#1

blackjack4494 committed:
updated angular-websockets && is now working again

blackjack4494 committed:
Update README.md

changed repo

blackjack4494 committed:
angular version reverted -hotfix-

blackjack4494 committed:
Merge remote-tracking branch 'origin/master'

kd7lxl commented on README.md:
```diff
@@ -9,23 +9,21 @@ useful diagnostic tool while making changes to your own network configuration.
 ## Example
 
-http://trace.hamwan.net/
+~~http://trace.hamwan.net/~~
```
I fixed this example site. No need to cross it out.

kd7lxl commented on README.md:
````diff
@@ -9,23 +9,21 @@ useful diagnostic tool while making changes to your own network configuration.
 
 ```sh
-gunicorn -k flask_sockets.worker mtr-web:app
+gunicorn -b 0.0.0.0 -k flask_sockets.worker mtr-web:app
````
I don't think it's good to bind all addresses in the usage example without any explanation of what this does. The link following this command goes to http://127.0.0.1:8000, which doesn't require usage of `-b 0.0.0.0`. I think it would be best to just drop this change.

kd7lxl commented on index.html:
```diff
@@ -82,7 +82,11 @@
+            url.protocol = url.protocol.replace('http', 'ws');
+            url.href = url.href + "mtr"
+            ws = $websocket(url.href);
+            //console.log("log: " + url.href);
```
I like this protocol replacement! But can you please remove the commented out `console.log()` line?

kd7lxl reviewed (changes requested):
Thank you for your contribution to this project! I like the code changes, but not all of the readme changes are necessary. Since they are not separate commits, I can't easily cherry-pick them. Can you please make the requested changes?

kd7lxl commented:
I merged forward3d/mtr-web@e105ba4 (protocol replacement). Please rebase your PR against master to incorporate this change.

blackjack4494 committed:
prep for new mtr

blackjack4494 committed:
Update README.md

blackjack4494 committed:
changed to float since new mtr

blackjack4494 committed:
float mtr added

blackjack4494 committed:
Update README.md

blackjack4494 committed:
Update README.md

blackjack4494 commented:
> I merged [forward3d/mtr-web@e105ba4](https://github.com/forward3d/mtr-web/commit/e105ba4) (protocol replacement). Please rebase your PR against master to incorporate this change.

I am already a step further.
I pulled the mtr source code and modified it. Now my mtr outputs way more when using -p
I am still figuring out how to add ASN as well.
You can have a look
https://github.com/blackjack4494/mtr-web
there is a test site. Also the modified mtr is included. Otherweise you can pull my mtr source and build it yourself.

blackjack4494 committed:
Update README.md

blackjack4494 committed:
Update README.md

since requested added domain and ipv6 support

blackjack4494 committed:
Update README.md

kd7lxl commented:
If you submit each new feature in its own PR, I will evaluate each for its merits. One thing I am not interested in merging is a pre-compiled binary. I don't think this is best practice and would rather provide a reproducible build script for any required binary.

blackjack4494 commented:
> If you submit each new feature in its own PR, I will evaluate each for its merits. One thing I am not interested in merging is a pre-compiled binary. I don't think this is best practice and would rather provide a reproducible build script for any required binary.

Sure I understand that. I thought about the same. But just including it was the fastest way. Compiling mtr yourself can be a bit annoying. As I said the source of the mtr is also available. I linked it in my fork.

blackjack4494 committed:
first steps server/client

separate server and client
freely choose input (requires running mtr-srv on given address)

blackjack4494 committed:
hotfix

forgot ':' in mtr-srv
added ws protocol for wsaddr
added logging

blackjack4494 committed:
added server/client version readme

blackjack4494 committed:
24/7 service with screen

blackjack4494 committed:
Create roadmap.md

kd7lxl commented:
I'm closing this since your branch has diverged from the original description of the PR. If you would like me to consider merging your changes, please create separate PRs for each new fix or feature. You will need to create separate feature branches in your repo for this to work.
